src/backend/distributed/planner/query_pushdown_planning.c

```diff
@@ -534,11 +534,6 @@ SubqueryMultiNodeTree(Query *originalQuery, Query *queryTree,
 		RaiseDeferredError(unsupportedQueryError, ERROR);
 	}
 
-	/*
-	 * In principle, we're first trying subquery pushdown planner. If it fails
-	 * to create a logical plan, continue with trying the single table
-	 * repartition subquery planning.
-	 */
 	DeferredErrorMessage *subqueryPushdownError = DeferErrorIfUnsupportedSubqueryPushdown(
```

Comment:
then, why do we have if/else?  Do we have rErrorIfUnsupportedSubqueryPushdown() equivalent such that we only call that?

Reply:
yeah we don't need that `else` actually, fixing now